src/c++/perf_analyzer/genai-perf/tests/test_cli.py

```diff
@@ -210,9 +210,33 @@ def test_load_level_mutually_exclusive(self, monkeypatch, capsys):
         captured = capsys.readouterr()
         assert expected_output in captured.err
 
+    def test_compare_mutually_exclusive(self, monkeypatch, capsys):
```

Comment:
Would you be able to add sections headers for regular vs compare subcommand, if we're going that route?

Reply:
Sorry what is sections headers?

Reply:
Just like a multi-line comment at the start of each test header. That can help visually separate the tests. Something like this: https://github.com/triton-inference-server/triton_cli/blob/58ba74c4156b24541fd1fd6f462fe17f019d48d9/src/triton_cli/parser.py#L101-L103

Reply:
Added section header.

Reply:
Thanks! This might have not gotten pushed?